### PR TITLE
Update mxnet-federated requirements

### PR DESCRIPTION
**Pull request by danielnugraha**
### Description

Pyproject.toml and requirements.txt in mxnet-from-centralized-to-federated example is not synced.

### Explanation

Sync pyproject.toml and requirements.txt in mxnet-from-centralized-to-federated example and remove redundant numpy dependency.

### Checklist

- [x] Implement proposed change
- [x] Write tests
- [x] Update [documentation](https://flower.dev/docs/writing-documentation.html)
- [x] Update [changelog](https://github.com/adap/flower/blob/main/doc/source/changelog.rst)
- [x] Make CI checks pass
- [x] Ping maintainers on [Slack](https://flower.dev/join-slack/) (channel `#contributions`)

### CODE REVIEW

**Comment by danielnugraha** [comment]
Tested on python 3.8 and 3.10.

File: examples/mxnet-from-centralized-to-federated/pyproject.toml [hunk 1]
```diff
@@ -11,6 +11,4 @@ authors = ["The Flower Authors <hello@flower.dev>"]
 [tool.poetry.dependencies]
 python = ">=3.8,<3.11"
 flwr = ">=1.0,<2.0"
-# flwr = { path = "../../", develop = true }  # Development
 mxnet = "1.6.0"
-numpy = "1.23.1"
```
Comment (jafermarq) [inline examples/mxnet-from-centralized-to-federated/pyproject.toml:16]: we need to keep the `numpy = "1.23.1"`

File: examples/mxnet-from-centralized-to-federated/requirements.txt [hunk 2]
```diff
@@ -1,3 +1,2 @@
 flwr>=1.0,<2.0
 mxnet==1.6.0
-numpy==1.23.1
```
Comment (jafermarq) [inline examples/mxnet-from-centralized-to-federated/requirements.txt:3]: we need to keep the `numpy = "1.23.1"`

**Review by jafermarq — COMMENTED** [review]
It works on x86, but we need to keep the `numpy` to `1.23`

**Comment by jafermarq** [comment]
Closed in favour of #2724

**Pull request closed without merging**